packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -61,6 +62,30 @@ const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
 	);
 };
 
+interface TrackDesignViewProps {
+	categorization?: string | null;
+	design: Design;
+	isPremiumThemeAvailable?: boolean;
+}
+
+const TrackDesignView: React.FC< TrackDesignViewProps > = ( {
+	categorization,
+	design,
+	isPremiumThemeAvailable,
+} ) => {
+	useEffect( () => {
+		recordTracksEvent( 'calypso_design_picker_design_display', {
+			category: categorization,
+			design_type: design.design_type,
+			is_premium: design.is_premium,
+			is_premium_available: isPremiumThemeAvailable,
+			slug: design.slug,
+		} );
+	} );
```

Comment:
you have to use `design.slug` or an empty array as deps so that the event won't fire on every re-rendering